--- libquicktime-1.0.0/utils/gtk/lqt_gtk.h.orig	2007-10-28 13:31:56.009323146 +0000
+++ libquicktime-1.0.0/utils/gtk/lqt_gtk.h	2007-10-28 13:36:01.971800028 +0000
@@ -21,11 +21,11 @@
   lqt_parameter_info_t * parameter_info;
 
   int selected;
-  GtkTooltips * tooltips;
+  GtkTooltip * tooltips;
   } LqtGtkParameterWidget;
 
 LqtGtkParameterWidget *
-lqtgtk_create_parameter_widget(lqt_parameter_info_t * info, GtkTooltips * tooltips);
+lqtgtk_create_parameter_widget(lqt_parameter_info_t * info, GtkTooltip * tooltips);
 
 void lqtgtk_destroy_parameter_widget(LqtGtkParameterWidget*);
 
@@ -47,7 +47,7 @@
 
   lqt_parameter_info_t * parameter_info;
   int num_parameters;
-  GtkTooltips * tooltips;
+  GtkTooltip * tooltips;
   } LqtGtkCodecConfigWidget;
 
 LqtGtkCodecConfigWidget *
--- libquicktime-1.0.0/utils/gtk/lqt_gtk.c.orig	2007-10-28 13:32:01.008560480 +0000
+++ libquicktime-1.0.0/utils/gtk/lqt_gtk.c	2007-10-28 13:42:59.408117502 +0000
@@ -164,9 +164,7 @@
   //  GtkTooltips *tooltips = (GtkTooltips *)data;
 
   if(GTK_IS_BUTTON (widget))
-    gtk_tooltips_set_tip(w->tooltips, widget,
-                         w->parameter_info->help_string,
-                         NULL);
+    gtk_widget_set_tooltip_text(widget, w->parameter_info->help_string);
   }
 
 static void
@@ -181,7 +179,7 @@
 
 
 LqtGtkParameterWidget *
-lqtgtk_create_parameter_widget(lqt_parameter_info_t * info, GtkTooltips * tooltips)
+lqtgtk_create_parameter_widget(lqt_parameter_info_t * info, GtkTooltip * tooltips)
   {
   int i;
   LqtGtkParameterWidget * ret = calloc(1, sizeof(LqtGtkParameterWidget));
@@ -197,7 +195,7 @@
         {
         ret->widget = gtk_check_button_new_with_label(info->real_name);
         if(info->help_string)
-          gtk_tooltips_set_tip(ret->tooltips, ret->widget, info->help_string, info->help_string);
+	  gtk_widget_set_tooltip_text(ret->widget, info->help_string);
         }
       /* Integer with limits -> slider */
       else if(info->val_min.val_int < info->val_max.val_int)
@@ -216,7 +214,7 @@
         
         gtk_scale_set_digits(GTK_SCALE(ret->widget), 0);
         if(info->help_string)
-          gtk_tooltips_set_tip(ret->tooltips, ret->widget, info->help_string, info->help_string);
+	  gtk_widget_set_tooltip_text(ret->widget, info->help_string);
         }
       /* Spinbutton */
       else
@@ -233,7 +231,7 @@
                                           0.0,
                                           0);
         if(info->help_string)
-          gtk_tooltips_set_tip(ret->tooltips, ret->widget, info->help_string, info->help_string);
+	  gtk_widget_set_tooltip_text(ret->widget, info->help_string);
         }
       break;
     case LQT_PARAMETER_FLOAT:
@@ -260,7 +258,7 @@
                                 GTK_POS_LEFT);
         
         if(info->help_string)
-          gtk_tooltips_set_tip(ret->tooltips, ret->widget, info->help_string, info->help_string);
+	  gtk_widget_set_tooltip_text(ret->widget, info->help_string);
         }
       /* Spinbutton */
       else
@@ -281,7 +279,7 @@
                                    info->num_digits);
 
         if(info->help_string)
-          gtk_tooltips_set_tip(ret->tooltips, ret->widget, info->help_string, info->help_string);
+	  gtk_widget_set_tooltip_text(ret->widget, info->help_string);
         }
       break;
     case LQT_PARAMETER_STRING:
@@ -289,7 +287,7 @@
       gtk_misc_set_alignment(GTK_MISC(ret->label), 0.0, 0.5);
       ret->widget = gtk_entry_new();
       if(info->help_string)
-        gtk_tooltips_set_tip(ret->tooltips, ret->widget, info->help_string, info->help_string);
+	gtk_widget_set_tooltip_text(ret->widget, info->help_string);
       break;
     case LQT_PARAMETER_STRINGLIST:    /* String with options */
       ret->selected = 0;
@@ -343,7 +341,7 @@
 
 static GtkWidget * create_table(lqt_parameter_info_t * parameter_info,
                                 LqtGtkParameterWidget ** widgets,
-                                int num_parameters, GtkTooltips * tooltips)
+                                int num_parameters, GtkTooltip * tooltips)
   {
   int i;
   GtkWidget * ret;
