Index: libquicktime-1.2.2/plugins/ffmpeg/lqt_ffmpeg.c
===================================================================
--- libquicktime-1.2.2.orig/plugins/ffmpeg/lqt_ffmpeg.c
+++ libquicktime-1.2.2/plugins/ffmpeg/lqt_ffmpeg.c
@@ -1031,8 +1031,8 @@ static void set_codec_info(struct CODECI
   snprintf(ffmpeg_long_name, 256, "%s", map->name);
   snprintf(ffmpeg_description, 256, "%s", map->name);
   
-  if((map->encoder && (map->encoder->type == CODEC_TYPE_VIDEO)) ||
-     (map->decoder && (map->decoder->type == CODEC_TYPE_VIDEO))){
+  if((map->encoder && (map->encoder->type == AVMEDIA_TYPE_VIDEO)) ||
+     (map->decoder && (map->decoder->type == AVMEDIA_TYPE_VIDEO))){
        codec_info_ffmpeg.type = LQT_CODEC_VIDEO;
   } else {
        codec_info_ffmpeg.type = LQT_CODEC_AUDIO;
