--- libquicktime-1.0.2/m4/avcodec.m4.orig	2005-10-28 14:16:40.000000000 +0200
+++ libquicktime-1.0.2/m4/avcodec.m4	2008-08-27 11:03:04.000000000 +0200
@@ -12,7 +12,7 @@
 avcodec_ok="false"
 AC_TRY_RUN([
     #include <stdio.h>
-    #include <ffmpeg/avcodec.h>
+    #include <libavcodec/avcodec.h>
     int main()
     {
     FILE * output;
--- libquicktime-1.0.2/plugins/ffmpeg/ffmpeg.h~	2007-09-26 20:36:05.000000000 +0200
+++ libquicktime-1.0.2/plugins/ffmpeg/ffmpeg.h	2008-08-27 11:18:03.000000000 +0200
@@ -26,7 +26,7 @@
 #define QUICKTIME_FFMPEG_H
 
 #include <quicktime/qtprivate.h>
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 
 void quicktime_init_video_codec_ffmpeg(quicktime_video_map_t *vtrack,
                                        AVCodec *encoder, AVCodec *decoder);
--- libquicktime-1.0.2/plugins/ffmpeg/params.c~	2007-09-26 20:36:05.000000000 +0200
+++ libquicktime-1.0.2/plugins/ffmpeg/params.c	2008-08-27 11:18:18.000000000 +0200
@@ -24,7 +24,7 @@
 
 #include "lqt_private.h"
 #include "params.h"
-#include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 #include <string.h>
 
 typedef struct
--- libquicktime-1.0.2/plugins/ffmpeg/video.c~	2007-12-19 10:36:03.000000000 +0100
+++ libquicktime-1.0.2/plugins/ffmpeg/video.c	2008-08-27 11:23:21.000000000 +0200
@@ -32,7 +32,7 @@
 #define LOG_DOMAIN "ffmpeg_video"
 
 #ifdef HAVE_LIBSWSCALE
-#include <swscale.h>
+#include <libswscale/swscale.h>
 #endif
 
 // Enable interlaced encoding (experimental)
